### PR TITLE
Setup docker + images #24

### LINKED ISSUE #24: Setup docker + images
- Study docker and how to setup monbodb image.
- Look into mongodb clients 
- write documentation onto how to setup docker and mongo 

https://docs.docker.com/desktop/install/windows-install/

### PR DESCRIPTION
**Pull request by Patricijia**
Steps to setup the docker and MongoDB

### CODE REVIEW

**Review by Lucasmeteenc — DISMISSED** [review]
Clear and good explanation

File: README.md [hunk 1]
```diff
@@ -1,9 +1,69 @@
...
 In the terminal run `cd frontend`, to access the frontend app. To start the web app run `npm start`. The web app should open a tab automatically, but if this is not the case it is located at http://localhost:3000/. To stop the app, press `Ctrl + c` in the terminal and answer `y` to the prompt (if there is one).   
+
+### Setting up DataBase: MongoDB + Docker
+We will use Docker to host a MongoDB instance locally.
+
+- [ ] Install [Docker Desktop](https://www.docker.com/products/docker-desktop/). Run it.
+
+- [ ] Open Command Prompt (cmd). 
+
+- [ ] Enter `docker pull mongo: latest`
```
Comment (VictorVader) [inline README.md]: For me 'docker pull mongo' worked (cmd gave me this error message "docker pull" requires exactly 1 argument). The default tag for this command, 'latest', was automatically used

File: README.md [hunk 2]
```diff
@@ -1,9 +1,69 @@
...
+
+- [ ] Enter `docker images`. You should see something similar:
+
+```
+REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
+mongo        latest    a440572ac3c1   3 weeks ago   639MB
+```
+
+- [ ] Create folder "mangodb" in your root directory e.g. C:\Users\20193308.
```
Comment (Lucasmeteenc) [inline README.md]: Should me mongodb?

Comment (Patricijia) [reply README.md]: Haha yes

File: README.md [hunk 3]
```diff
@@ -1,9 +1,69 @@
...
+C:\Users\20193308\mongodb> docker ps
+
+CONTAINER ID   IMAGE          COMMAND                  CREATED          STATUS          PORTS                      NAMES
+a56ffa709baa   mongo:latest   "docker-entrypoint.s…"   7 seconds ago    Up 7 seconds    0.0.0.0:2717->27017/tcp    Bumbal
+```
+
+- [ ]  Enter `mongosh`. You should see something similar:
+
+```
+C:\Users\20193308\mongodb> mongosh
```
Comment (Lucasmeteenc) [inline README.md:53]: Doesn't work for me (I think because i am on windows) probably need to install mongosh

Comment (Patricijia) [reply README.md:53]: Strange, have you tried to open the Docker Desktop?

Comment (Lucasmeteenc) [reply README.md:53]: image
yes

**Review by Lucasmeteenc — DISMISSED** [review]
This works!

**Review by VictorVader — APPROVED** [review]

**Review by VictorVader — APPROVED** [review]

**Review by VictorVader — APPROVED** [review]

**Review by Lucasmeteenc — APPROVED** [review]
image

as ordered

**Review by VictorVader — APPROVED** [review]

**Pull request merged**